.github/workflows/ci.yml

```diff
@@ -85,7 +85,7 @@ jobs:
             rustc: nightly
           - name: "Unit Tests"
             cmd: test
-            args: --features multitenant,analytics,geoblock,functional_tests
+            args: --features multitenant,analytics,geoblock,functional_tests,apns_tests,fcm_tests,fcmv1_tests
```

Comment:
Made into separate features to enable running tests locally without all of these test secrets.